cypress/support/commands.js

```diff
@@ -39,3 +43,51 @@ Cypress.Commands.add('register', (email = 'riot@qa.team', username = 'riot', pas
     password
   });
 });
+
+Cypress.Commands.add('login', (email = 'riot@qa.team', username = 'riot', password = '12345Qwert!') => {
+    cy.request('POST', '/users/login', {
+      user: {
+        email,
+        username,
+        password,
+      },
+    }).then((response) => {
+      const user = {
+        bio: response.body.user.bio,
+        effectiveImage:
+          'https://static.productionready.io/images/smiley-cyrus.jpg',
+        email: response.body.user.email,
+        image: response.body.user.image,
+        token: response.body.user.token,
+        username: response.body.user.username,
+        id: response.body.user.id
+      };
+      window.localStorage.setItem('user', JSON.stringify(user));
+      cy.setCookie('drash_sess', response.body.user.token);
+    });
+  }
+);
+
+Cypress.Commands.add('createArticle', (title, description, body) => {
+  cy.getCookie('drash_sess').then((token) => {
+    const authToken = token.value;
```

Comment:
response is not defined. You should define it within the function or pass it as a parameter in order to use it.